### ISSUE TITLE
New non-fullscreen renderer partially overwrites message line in Vim

### ISSUE THREAD

**Issue by toiffel** [junegunn/fzf#814]
- Category
    - [x] fzf binary
    - [ ] fzf-tmux script
    - [ ] Key bindings
    - [ ] Completion
    - [x] Vim
    - [ ] Neovim
    - [ ] Etc.
- OS
    - [x] Linux
    - [x] Mac OS X
    - [ ] Windows
    - [ ] Windows Subsystem for Linux
    - [ ] Etc.
- Shell
    - [ ] bash
    - [ ] zsh
    - [ ] fish

Steps to reproduce:

1. Run Vim
2. Create some mapping for `:FZF` command with `<silent>` option, e.g. `:nnoremap <silent> <Leader>f :FZF<CR>`
3. Issue any message with sufficient length, e.g. `:echo "fzfrockzzzzzzzzzz"`
4. Press `<Leader>f` to run FZF

Expected: FZF starts and fully overwrites the message with file list. Alternatively, it keeps the message and shows the file list under it.
Observed: FZF overwrites the message only partially, leaving some characters from message on the screen.

![fzf](https://cloud.githubusercontent.com/assets/1043953/22118217/19f21b5c-dea1-11e6-808e-1de0eaae6636.png)

Here `Setup.hs` is the first file in list and `zzzzzzz` is what left from the message line.

The strange thing is that this happens only with `<silent>` option. Non-silent mapping works well.

**Comment by junegunn** [comment]
Added `tput el` to clear the last line before starting fzf as the workaround for now. I'll make the next version of fzf do it natively, so that it is no longer needed.
